buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -169,19 +169,23 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                     }
 
                                     // Clean up defunct pipelines workspaces
-                                    dir("${buildWorkspace}/../") {
-                                        def retStatus = 0
+                                    def retStatus = 0
+                                    def cleanWSDirs = get_other_workspaces("${buildWorkspace}/../")
+
+                                    if (cleanWSDirs) {
+                                        def cleanWSDirsStr = "${buildWorkspace}/../"
+                                        cleanWSDirsStr += cleanWSDirs.join(" ${buildWorkspace}/../")
```

Comment:
This looks to me like it would duplicate the `${buildWorkspace}/../` for every item in `cleanWSDirs` which seems like an odd choice.  Is that correct (and deliberate)?

Reply:
correct

Reply:
Please explain what this is doing.

Reply:
please see https://github.com/eclipse/openj9/pull/5618#discussion_r279072085 below.
`cleanWSDirs` is a list of directories names representing workspaces of past builds left over by the clean workspace plugin failures (cleanWs) or pipelines builds that do no clean their workspaces after they run. 
`cleanWSDirsStr` is a string (space separated) of past builds workspaces full paths.
To avoid failures on windows, `rm -fr ${cleanWSDirsStr}` command runs in the build workspace aka current directory.

Reply:
The delete in the current directory does not work on windows. 
eg 
```
dir("${buildWorkspace}/../") {
   sh "rem -fr ..."
}
```
Deleting the directories in one by one in a `for` loop  is time consuming.